### PR TITLE
UI: Technology mobile responsive

### PR DESCRIPTION
**Pull request by austin-pham-goldenowl**
- apply mixins 
- mobile responsive for Technology Page

### CODE REVIEW

File: src/components/Tabs/TechTabs.js [hunk 1]
```diff
@@ -30,7 +30,9 @@ class TabContainer extends Component {
               <div className="col-md-6 ml-auto right-side">
                 {FrontEndTechs.map((tech) => (
                   <div key={tech.name} className="tech-tab__logo-wrapper">
-                    <img src={tech.url} alt={tech.name} />
+                    <div>
```
Comment (hansanwok) [inline src/components/Tabs/TechTabs.js]: redundant div???

File: src/components/Tabs/TechTabs.js [hunk 2]
```diff
@@ -55,7 +57,9 @@ class TabContainer extends Component {
               <div className="col-md-6 ml-auto right-side">
                 {BackEndTechs.map((tech) => (
                   <div key={tech.name} className="tech-tab__logo-wrapper">
-                    <img src={tech.url} alt={tech.name} />
+                    <div>
```
Comment (hansanwok) [inline src/components/Tabs/TechTabs.js]: redundant div???

File: src/components/Tabs/TechTabs.js [hunk 3]
```diff
@@ -81,7 +85,9 @@ class TabContainer extends Component {
               <div className="col-md-6 ml-auto right-side">
                 {MobileStackTechs.map((tech) => (
                   <div key={tech.name} className="tech-tab__logo-wrapper">
-                    <img src={tech.url} alt={tech.name} />
+                    <div>
```
Comment (hansanwok) [inline src/components/Tabs/TechTabs.js]: redundant div???

File: src/components/Tabs/TechTabs.js [hunk 4]
```diff
@@ -107,7 +113,9 @@ class TabContainer extends Component {
...
```
Comment (hansanwok) [inline src/components/Tabs/TechTabs.js]: redundant div???

File: src/components/Tabs/TechTabs.js (hunk 1 above)
Comment (austin-pham-goldenowl) [reply src/components/Tabs/TechTabs.js]: it's an image wrapper

**Pull request merged**